cruise-control/src/yaml/endpoints/removeDisks.yaml

```
@@ -0,0 +1,69 @@
+RemoveDisksEndpoint:
+  post:
+    operationId: removeDisks
+    summary: Move all replicas from the specified disk to another disk of the same broker.
```

```suggestion
    summary: Move all replicas from the specified disk to other disks of the same broker.
```

It's not only one disk where we move the replicas to, right?